knowledge-base/grid-transposed.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Transpose Grid
+description: Learn how to show transposed Grid data in a table or a Form. The transposed data may or may not support editing.
+type: how-to
+page_title: How to Transpose the Telerik Grid for Blazor
+slug: grid-kb-transposed
+tags: telerik, blazor, grid, transposed
+ticketid: 1569198, 1639288, 1660386
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to transpose the data Grid component?
```

Comment:
Something is off here. Should this be "the data of the Grid component" or "the Data Grid component"?